input/dispatch/dispatch.go

```diff
@@ -14,7 +14,7 @@ func Get(desire def.Input) input.Input {
 	case "dir":
 		input = dir.New(desire)
 	case "tar":
-		input = tar.New(desire)
+		input = tar2.New(desire)
```

Comment:
`exec-tar` would work. Don't forget to change the name so that it doesn't panic.